variables/_svgs.pcss

```diff
@@ -354,6 +354,24 @@
 	}
 }
 
+/* -----------------------------------------------------------------------------
+ * SVG - Plus
+ * ----------------------------------------------------------------------------- */
+
+@svg-load plus url('../icons/plus.svg') {
+
+	path {
+		stroke: var(--color-icon-focus);
+	}
+}
+
+@svg-load plus-focus url('../icons/plus.svg') {
```

Comment:
you have the wrong one changed to plus-focus

Reply:
right, we're following the var name. We already discussed this.